codalab/server/rest_server.py

```diff
@@ -134,7 +136,10 @@ def wrapper(*args, **kwargs):
                     raise
                 code, message = exception_to_http_error(e)
                 if code == INTERNAL_SERVER_ERROR:
-                    traceback.print_exc()
+                    pretty_json = formatting.pretty_json(request.json) if request.json is not None else "<EMPTY>"
```

Comment:
maybe use '<none>' to be consistent with the convention for rendering None in this project